tests/cases/standalone/common/system/information_schema.result

```diff
@@ -31,25 +31,43 @@ where table_catalog = 'greptime'
   and table_schema != 'public'
```

Comment:
Considering the purpose of the test, just add a filter condition `table_schema != 'information_schema'`
```suggestion
  and table_schema != 'public'
  and table_schema != 'information_schema'
```